src/main/java/com/exasol/adapter/document/mapping/SourceReferenceColumnMapping.java

```diff
@@ -0,0 +1,60 @@
+package com.exasol.adapter.document.mapping;
+
+import com.exasol.adapter.metadata.DataType;
+
+/**
+ * This class describes a column that maps the source reference. For example in the files-virtual-schemas the name of
+ * the file that contained the document. The name of this column is always {@code SOURCE_REFERENCE}, except from if it
+ * is is used as foreign key (not yet possible).
+ */
```

Comment:
There is extra `is` before used. Could you also add `<p>` after the first sentence?